lib/jsdom/living/domparsing/XMLSerializer.webidl

```diff
@@ -0,0 +1,5 @@
+[Constructor,
```

Comment:
Could you add a URL to where the IDL is found?

Reply:
It’s&nbsp;at: <https://w3c.github.io/DOM-Parsing/#the-xmlserializer-interface>
```suggestion
// https://w3c.github.io/DOM-Parsing/#the-xmlserializer-interface
[Constructor,
```